editor.planx.uk/src/@planx/components/AddressInput/AddressInput.stories.tsx

```diff
@@ -0,0 +1,55 @@
+import { Meta, StoryObj } from "@storybook/react";
+import { userEvent, within } from "@storybook/testing-library";
+import React from "react";
+
+import Wrapper from "../fixtures/Wrapper";
+import Editor from "./Editor";
+import Public from "./Public";
+
+export default {
+  title: "PlanX Components/AddressInput",
+  component: Public,
+} as Meta;
+
+export const EmptyForm: StoryObj = {
```

Comment:
What the TS support like here? Do we have an option to do anything like `StoryObj<AddressInputProps>` or maybe `StoryObj<typeof AddressInputProps>` for additional safety / future proofing here?

If there's a way of doing this it could be a neat way of ensuring we keep our stories up to date when component inputs and outputs change.

Happy for this to be a follow on task or I can pick it up also 👍

Reply:
TS support & more proactive typing is as good question here - there's some docs in this direction here (be careful to sift out CSF3 format though which we're on now): https://storybook.js.org/docs/react/api/csf#args-story-inputs

I can update to this: 
```
export const EmptyForm: StoryObj<AddressInput> = {
  args: {
    title: "Enter your address",
    description: "It might be where you live",
  },
};
```

But, `args` property has type `args?: Partial<AddressInput> | undefined` - so it unfortunately doesn't complain or throw any errors/warnings if I, for example, remove `title` which we say is required.

One to keep thinking on?

Reply:
Ah I see - lets come back to this then - happy with it as it just now! 🙂